includes/dbFacile.php

```diff
@@ -114,9 +118,13 @@ function dbUpdate($data, $table, $where = null, $parameters = array()) {
 
 	// need field name and placeholder value
 	// but how merge these field placeholders with actual $parameters array for the WHERE clause
-	$sql = 'UPDATE `' . $table . '` set ';
+	$sql = 'UPDATE `' . $table . '` SET ';
 	foreach($data as $key => $value) {
-                $sql .= "`".$key."` ". '=:' . $key . ',';
```

Comment:
Think we should fix the query rather than alter dbFacile. Do you know where the query was being called from that is broke?